conf/application.conf

```
@@ -72,6 +72,7 @@ contact-frontend.serviceId = "AddressLookupFrontend"
 
 microservice {
   hosts.allowList = ["tax.service.gov.uk", "www.tax.service.gov.uk", "admin.tax.service.gov.uk"]
+  selectPageConfig.showNoneOfTheseOption = true
```

```suggestion
  selectPageConfig.showNoneOfTheseOption = true
```

I think I would leave this `false` for now. To allow teams time to switch up their test implementations.